demo/index.html

```diff
@@ -26,6 +26,8 @@
       --downloadButtonBg: #547fba;
       --externalButtonColor: #547fba;
       --externalButtonBg: #fff;
+      --closeMenuIconWidth: 18px;
```

Comment:
we should go with a standard padding on the right and remove this extra css var.